docs/api.md

```diff
@@ -85,6 +85,22 @@ Set a custom default layout used for preview index and individual previews:
 
     config.view_component.default_preview_layout = "component_preview"
 
+### #generate_locale
+
+Always generate translation file alongside the component:
+
+    config.view_component.generate_locale = true
+
+Defaults to `false`.
+
+### #generate_splitted_locale_files
```

Comment:
What about renaming the option to use the word `distinct`, as you did in the generators docs? It feels clearer to me.

```suggestion
### #generate_distinct_locale_files
```